tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/RedeployTest.java

```diff
@@ -610,9 +610,24 @@ public void testRedeployBridge() throws Exception {
             Queue queue = session.createQueue("c-from");
             MessageProducer producer = session.createProducer(queue);
             producer.send(session.createMessage());
-            Wait.assertEquals(1, () -> embeddedActiveMQ.getActiveMQServer().locateQueue("c-to").getMessageCount());
+            Wait.assertEquals(1, () -> embeddedActiveMQ.getActiveMQServer().locateQueue("c-to").getMessageCount(), 2000);
          }
 
+         brokerXML.toFile().setLastModified(System.currentTimeMillis() + 1000);
+         latch.setCount(1);
+         embeddedActiveMQ.getActiveMQServer().getReloadManager().setTick(tick);
+         latch.await(10, TimeUnit.SECONDS);
+         Thread.sleep(5000);
```

Comment:
These sleeps are a bit trouble on the extended testsuite... 

If you need some condition to happen, I would recommend some Wait.assert clause... and only waiting the required time.  or making retries faster.